geoshop/entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -euo pipefail
+
+cd /geoshop/geoshop-back
+
+if [[ "$( python3 manage.py showmigrations | grep '\[ \]' )" != "" ]]; then
+    python3 manage.py migrate --noinput
+else
+    echo "No pending migrations"
+fi
+
+if [[ ! -f "/geoshop/.fixtures_lock" ]]; then
+    python3 manage.py fixturize
+    if [[ $? == 0 ]]; then 
+      touch "/geoshop/.fixtures_lock" 
+    fi
+else
+    echo "No fixturizing needed."
+fi
+
+if [[ ! -f "/geoshop/.sampledata_lock" ]]; then
+    # TODO: Pass host, password and dbname from the env section of the postgis
+    PGPASSWORD=geoshop psql --host=postgis --username=geoshop --dbname=geoshop -p 5432 < /geoshop/sampledata.sql
```

Comment:
We will have to keep in mind here that if we ever add a new migration we the data model of this data model in `sampledata.sql`  will also need to be upgraded/changed.
Bu I think for the moment it is the best solution.